spec/jobs/ingest_job_spec.rb

```diff
@@ -65,7 +67,8 @@
         shelve: true
       },
       access: {
-        access: 'citation-only'
+        access: 'stanford',
+        download: 'stanford'
```

Comment:
would we in any way be better served keeping this citation only or the new equivalent and making a separate test for stanford access?

Reply:
`citation-only` isn't a valid value for a file. The actual values don't matter, as long as they're valid.